sycl/doc/OptionalDeviceFeatures.md

```diff
@@ -629,11 +629,122 @@ example, if a new device is released before there is a new DPC++ release.  In
 fact, the DPC++ driver supports a command line option which allows the user
 to select an alternate configuration file.
 
-**TODO**: More information will be inserted here when we merge
-[this separate PR][7] into this design document.
+**TODO**: 
+* The names of the devices in the configuration file.
+* Location of the default device configuration file
```

Comment:
I suggest you remove the TODO "The names of the devices in the configuration file" because @bader has already suggested you add a TODO about this in the section "AOT target identification".

Likewise, I suggest you move the TODO "Location of the default device configuration file" below to the section titled "Configuration file location and driver option".